### DISCUSSION TITLE
"nvchad_ui.lsp not found" after fresh install

### DISCUSSION THREAD

**Discussion by hexdigest** [NvChad/NvChad#2404] [Q&A] [answered]
Failed to run `config` for nvim-lspconfig
                             
/home/max/.config/nvim/lua/plugins/configs/lspconfig.lua:2: module 'nvchad_ui.lsp' not found:
^Ino field package.preload['nvchad_ui.lsp']                                  
cache_loader: module nvchad_ui.lsp not found
cache_loader_lib: module nvchad_ui.lsp not found

**Comment by siduck** [comment, marked as answer]
@hexdigest impossible, that isnt even a fresh install!  look, its nvchad.lsp here and not nvchad_ui.lsp

https://github.com/NvChad/NvChad/blob/v2.0/lua/plugins/configs/lspconfig.lua#L2

rm -rf .config/nvim ( backup custom dir tho ), then try
